homeassistant/components/zha/alarm_control_panel.py

```diff
@@ -94,6 +95,6 @@ async def async_alarm_trigger(self, code: str | None = None) -> None:
         self.async_write_ha_state()
 
     @property
-    def state(self) -> str | None:
+    def alarm_state(self) -> AlarmControlPanelEntityState | None:
         """Return the state of the entity."""
         return self.entity_data.entity.state["state"]
```

Comment:
This needs to change too. There needs to be a map between the Home Assistant enum and the library string.